openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1206,3 +1214,89 @@ def validate_glusterfsd_memory_usage(self, size_limit):
                     "Failed memory used  of glusterfsd {} is greater than the"
                     " expected size {} for node/pod {}".format(
                         memory_used, size_limit, gluster_node))
+
+    def heketi_volumes_in_batch(
+            self, heketi_client_node, heketi_server_url, vol_count, size=1,
+            batch_amount=8, timeout=3600, interval=10):
+        """Creates heketi volume in batches.
+
+        Args:
+            heketi_client_node (str): Node on which cmd has to be executed.
+            heketi_server_url (str): Heketi server url.
+            vol_count (int): Count of volumes to be create.
+            size (str): Volume size.
+            batch_amount (int): Amount of pods to be created in one batch.
+            timeout (int): Timeout for one batch.
+            wait_step ( int): Time interval between each interation.
+
+        Returns:
+            list: list of heketi volumes.
+        """
+        h_vols, async_obj = [], []
+
+        # Fetch heketi volume list before operation
+        h_vols_count_initial = len(heketi_volume_list(
+            heketi_client_node, heketi_server_url, json=True).get('volumes'))
+
+        # Temporary replace g.run with g.async_run in heketi_volume_create func
+        # to be able to run it in background.Also, avoid parsing the output as
```

Comment:
```suggestion
        # to be able to run it in background. Also, avoid parsing the output as
```
Add space before Also

Reply:
Done